source/includes/steps-starting-compass-individual-fields.yaml

```diff
@@ -141,10 +141,9 @@ content: |
                  Select :guilabel:`Kerberos` if the deployment uses
                  :manual:`Kerberos </core/kerberos/>` as
                  its authentication mechanism. If selected, you must
-                 provide the
-                 :manual:`Principal </core/kerberos/#principals>`,
-                 :guilabel:`Password`, and :guilabel:`Service Name` to
-                 authenticate the user.
+                 provide the :manual:`Principal </core/kerberos/#principals>`
+                 and :guilabel:`Service Name` to authenticate the user. 
+                 The :guilabel:`Password` field should be left blank. 
```

Comment:
[nit] Let's make this more direct. "Leave the :guilabel:`Password` field blank".